common/types/status.go

```diff
@@ -1,3 +1,4 @@
+// Package types is for types that are shared by multiple packages
```

Comment:
Move types in `common` to here.

Reply:
A lot of those types have functions associated with them. I'm not going to push the scope of this PR with those changes.